Mage.Sets/src/mage/cards/t/TrespassersCurse.java

```diff
@@ -100,7 +100,8 @@ public boolean checkTrigger(GameEvent event, Game game) {
         Permanent enchantment = game.getPermanent(this.getSourceId());
         if (enchantment != null
                 && enchantment.getAttachedTo() != null
-                && game.getControllerId(event.getSourceId()).equals(enchantment.getAttachedTo())) {
+                && game.getControllerId(event.getSourceId()).equals(enchantment.getAttachedTo())
+                && game.getPermanent(this.getSourceId()).isCreature()) {
```

Comment:
Are we sure this works? Trespasser's Curse now no longer triggers at all, even when the opponent brings in a creature. I think this merge broke it.